Source/include/Ably/ARTPush.h

```diff
@@ -73,6 +80,20 @@ NS_ASSUME_NONNULL_BEGIN
 /// See [iOS push notifications tutorial](https://ably.com/tutorials/ios-push-notifications#step7-register-push-ably) for details.
 + (void)didFailToRegisterForRemoteNotificationsWithError:(NSError *)error realtime:(ARTRealtime *)realtime;
 
+// Location Push Registration token
+
+/// See [iOS push notifications tutorial](https://ably.com/tutorials/ios-location-push-notifications) for details.
++ (void)didRegisterForLocationNotificationsWithDeviceToken:(NSData *)deviceToken rest:(ARTRest *)rest;
```

Comment:
Since saving additional tokens is done via updating token (`deviceUpdateRegistration`), error can be obtained in the `ARTPushRegistererDelegate.didAblyPushRegistrationFail:` delegate method. In case of error user need to call `activate` again (https://github.com/ably/ably-cocoa/blob/dacb68be999ed30db3f915f49f3449d3ef47ded6/Source/ARTPushActivationState.m#L252C26-L252C26).

Reply:
OK, so they can detect the failure case, but how about success ("Is there a way for the user to know that the location token has successfully been submitted to Ably?")

Reply:
https://github.com/ably/ably-cocoa/pull/1771/commits/9960e6466b354a0fe0ea4e1be28bb88629b02ef3 and https://github.com/ably/ably-cocoa/pull/1771/commits/7dc36cc477ab0a1880a5f206054f10ff8b084353